src/pytest_dbfixtures/factories/dynamodb.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2016 by Clearcode <http://clearcode.cc>
+# and associates (see AUTHORS).
+
+# This file is part of pytest-dbfixtures.
+
+# pytest-dbfixtures is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# pytest-dbfixtures is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with pytest-dbfixtures.  If not, see <http://www.gnu.org/licenses/>.
+import pytest
+
+from pytest_dbfixtures.executors import TCPExecutor
+from pytest_dbfixtures.port import get_port
+from pytest_dbfixtures.utils import try_import, get_process_fixture
+
+
+class JarPathException(Exception):
+    """We do not know where user has a jar file.
+    So, we want to tell him that he has to provide a path to this file.
+
+    We raise the exception when the ``jar_path`` param in ``dynamodb_proc``
+    function is ``None``."""
+    pass
+
+
+def dynamodb_proc(jar_path=None, host='localhost', port='?'):
+    """
+    DynamoDB process factory.
+
+    :param str jar_path: a path to jar file. A path cannot contain spaces
+        http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html?shortFooter=true
+    :param str host: hostname
+    :param int port: port
+
+    :return: function which makes a DynamoDB process
+    """
+    if not jar_path:
```

Comment:
Either require it, or provide ability to set it through config/ini option. (`--dynamodb-path`)